README.md

```diff
@@ -1,77 +1,60 @@
-- `BUILD_CERTIFICATE_BASE64`: The Base64 version of the Apple signing certificate to build your iOS application.
-- `P12_PASSWORD`: The password for the Apple signing certificate.
-- `BUILD_PROVISION_PROFILE_BASE64`: The Base64 version of the Apple provisioning profile to build your iOS application.
-- `KEYCHAIN_PASSWORD`: A password for the keychain that will be created on the runner instance.
+You can find a list of contributors in the [CONTRIBUTORS.md](/CONTRIBUTORS.md) file.
 
-Be sure that you update the name of the provisioning profile in the `Gymfile` and update the app name, bundle identifier, Xcode project name, paths, and other settings in the fastlane files when modifying the template to your needs!
 
-### Swift Package and Fastlane Update ACCESS_TOKEN
+## Open Source Contributions
 
-The [Swift Package and Fastlane Update workflow](https://github.com/PSchmiedmayer/ContinousDelivery/blob/main/.github/workflows/update.yml) requires an `ACCESS_TOKEN` secret: a GitHub Personal Access Token (PAT) allowing write access to the repository.
-We suggest using a bot account to create the access token. Using the PAT triggers the GitHub Actions in the create PR. [The GitHub documentation provides instructions on creating a PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). The [scrop of the token](https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps) can be limited to the `public_repo` scope for public repositories or the `repo` scrope for private repositories as well as the `workflow` scope.
-
-Removing the `token` input in the GitHub action workflow results in using the default `GITHUB_TOKEN` and the GitHub Action bot account that does not trigger any possible merge checks in the newly created PR.
-
-### Contributors
-
-This project is based on [ContinousDelivery Example by Paul Schmiedmayer](https://github.com/PSchmiedmayer/ContinousDelivery). You can find a list of contributors in the `CONTRIBUTORS.md` file.
+As part of this work, we made the following open source contributions:
+* https://github.com/apple/swift-openapi-runtime/pull/115
+* https://github.com/apple/swift-openapi-generator/pull/625
+* https://github.com/openai/openai-openapi/pull/311#issuecomment-2600521373
+* https://github.com/StanfordSpezi/SpeziViews/pull/49
+* https://github.com/StanfordSpezi/SpeziAccount/pull/82
+* https://github.com/StanfordSpezi/SpeziLLM/pull/64
+* https://github.com/StanfordSpezi/SpeziViews/pull/43
+* https://github.com/StanfordSpezi/SpeziLLM/pull/61
```

Comment:
We should add the following sections (contributing, license, our research, and footer image) at the bottom of the README like we have them in other repos, including the Biodesign Logo. You can also add a TUM logo next to it if you want to.

Example:
https://github.com/StanfordBDHG/PediatricAppleWatchStudy?tab=readme-ov-file#contributing